doc/STYLE_GUIDE.md

```diff
@@ -54,10 +55,12 @@
     your point, not as complete running programs. If a complete running program
     is necessary, include it as an asset in `assets/code-examples` and link to
     it.
-* When using underscores, asterisks and backticks please use proper escaping (**\\\_**, **\\\*** and **\\\`** instead of **\_**, **\*** and **\`**)
-* References to constructor functions should use PascalCase
-* References to constructor instances should be camelCased
-* References to methods should be used with parentheses: `socket.end()` instead of `socket.end`
+* When using underscores, asterisks and backticks please use proper escaping
+  (**\\\_**, **\\\*** and **\\\`** instead of **\_**, **\*** and **\`**).
+* References to constructor functions should use PascalCase.
+* References to constructor instances should be camelCased.
```

Comment:
Nit: `be camelCased` -> `use camelCase`

That will be consistent with the previous line and avoid awkwardly using `camelCase` as a verb.